filebeat/input/log/log.go

```diff
@@ -8,11 +8,14 @@ import (
 	"github.com/elastic/beats/filebeat/harvester"
 	"github.com/elastic/beats/filebeat/input/file"
 	"github.com/elastic/beats/libbeat/logp"
+
+	file_helper "github.com/elastic/beats/libbeat/common/file"
```

Comment:
Please use a name without underscores, and don't leave an empty line between this import and the ones for other beats packages.